.github/workflows/python_test.yml

```diff
@@ -59,6 +57,33 @@ jobs:
           timeout_minutes: 12
           max_attempts: 2
           command: make unit_tests async=false args="--splits ${{ matrix.splitCount }} --group ${{ matrix.group }}"
+  integration-tests:
```

Comment:
Ah, I see the split of the make targets. Hm. Maybe we can make the naming a bit more verbose on the non-api-keys required side, since I think by default (imo) I would assume `integration-*` would require api keys. 

Suggestion then: rename this to something like `integration-tests-no-api-keys`
And similarly in the makefile

Reply:
done